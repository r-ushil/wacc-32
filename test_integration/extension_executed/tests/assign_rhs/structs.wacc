begin
  struct StringBox {
    string s
  }
  StringBox box_id(StringBox value) is
    return value
  end
  StringBox x = call box_id(StringBox { s: "hello there!" });
  println x.s
end
