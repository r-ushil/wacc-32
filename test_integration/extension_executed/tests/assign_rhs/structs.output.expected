Compiler exit code: 0
--------- stdout ---------
hello there!
--------------------------
Exit code: 0
